closed/autoconf/toolchain-win.cmake.in

```diff
@@ -26,6 +26,7 @@ set(CMAKE_SYSTEM_PROCESSOR "@OPENJDK_TARGET_CPU@")
 set(tool_dir "@OPENJ9_TOOL_DIR@")
 set(CMAKE_C_COMPILER "${tool_dir}/cl" CACHE FILEPATH "")
 set(CMAKE_CXX_COMPILER "${tool_dir}/cl" CACHE FILEPATH "")
+set(CMAKE_J9VM_C_ALT_COMPILER "${tool_dir}/clang" CACHE FILEPATH "")
```

Comment:
This should be deferred to a separate pull request, or be updated once a final language name is chosen (see https://github.com/eclipse/openj9/pull/10499#discussion_r483142852).